src/v/storage/types.h

```diff
@@ -353,6 +356,8 @@ struct compaction_config {
     model::timestamp eviction_time;
     // remove one segment if log is > max_bytes
     std::optional<size_t> max_bytes;
+    // Cannot compact past this offset (i.e. for unresolved txn records)
+    model::offset max_compactable_offset;
```

Comment:
please add a comment if we allow to compact strictly less or less-or-equal than `max_compactable_offset`

Reply:
Will clarify. For the recod: "max compactable offset" to me means that it is the maximum value for which the condition (compactable) holds. "Cannot compact past this offset" also implies `>`